src/ducks/store.js

```diff
@@ -7,7 +7,7 @@ import ipc from './ipc';
 import { rootReducer } from './modules/root';
 
 const persistConfig = {
-  key: 'architect',
+  key: 'architect6.1',
```

Comment:
Why was this changed? This will reset all app state for current users, won't it? We should revert unless it was intentional.

Reply:
Agreed, this shouldn't change unless the store has changed in some other drastic way (doesn't look like it has to me).